pex/bin/pex.py

```diff
@@ -476,6 +476,15 @@ def configure_clp():
       callback=increment_verbosity,
       help='Turn on logging verbosity, may be specified multiple times.')
```

Comment:
I think we need to also modify the help message for `verbosity`. I remember over Slack you had thought `-vvvvvvvvv` != `PEX_VERBOSE=9`, but this line suggests to me that it resolves the same thing: https://github.com/pantsbuild/pex/pull/680/files#diff-178808e5641b15e64b2d09f744976f03R673.

If this is the case, then I recommend changing the message to something like `'Turn on logging verbosity, may be specified multiple times. This will override `--no-emit-warnings.'`.

Reply:
I don't agree. There is a cross-product doc issue here and it seems to me just mentioning PEX_VERBOSE in the context of the PEX_EMIT_WARNINGS and --emit-warnings docs is enough. If you turn on PEX_VERBOSE, you've already asked for stderr to be polluted _in general_. Specific polluting options like \*emit\* are just subsets.

Reply:
Okay that's convincing enough.